bears/python/BlackBear.py

```diff
@@ -0,0 +1,34 @@
+from coalib.bears.LocalBear import LocalBear
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class BlackBear(LocalBear):
+    LANGUAGES = {'Python', 'Python 2', 'Python 3'}
```

Comment:
well `'Python 2'` is wrong if it is Python 3.6 only.

probably put only `'Python 3.6'` in here?